### PR TITLE
use seissol v1.3/Rusanov fluxes/ vtkhdf/ extend tpv13

### PR DESCRIPTION
**Pull request by Thomas-Ulrich**

### CODE REVIEW

**Comment by Thomas-Ulrich** [comment]
I think it is worth updating to the vtkhdf output:
image
image
vs the older output:
image
image

**Comment by AliceGabriel** [comment]
Thank you Thomas! 
I agree about updating the output.
Question: how different is the output after switching the flux in this coarse mesh?

**Comment by Thomas-Ulrich** [comment]
the figure above shows a qualitative answer.

**Comment by AliceGabriel** [comment]
Do you mind plotting them in the same figure

**Comment by AliceGabriel** [comment]
But overall I agree- we should update this

**Comment by AliceGabriel** [comment]
> Do you mind plotting them in the same figure

File: tpv13/tpv13_training.geo [hunk 1]
```diff
@@ -32,7 +32,7 @@ Z0 = -42e3;
 // Create the domain as a box
 domain = newv; Box(domain) = {X0, Y0, Z0, X1-X0, Y1-Y0, -Z0};
 
-// Create the fault as a vertically dipping rectangle, centered in x at the hypocenter
+// Create the fault as a in the x-y plane, centered in x at the hypocenter
```
Comment (AliceGabriel) [inline tpv13/tpv13_training.geo:35]: messed up grammar, missing words

**Comment by Thomas-Ulrich** [comment]
Ok, it turns out the 
reader.UpdateTimeStep(5.) that we had first meant read at time simulation time 5+1
(makes not much sense but that what I got when testing).
Anyway, if I put reader.UpdateTimeStep(9.)  and compare with the file 'output/{prefix}-free-surface-4.vtkhdf', I get matching data:

image

**Comment by Thomas-Ulrich** [comment]
Latest changes:
- I realized that pv.Plotter() needs to be closed to avoid memory leaks (leading to the RAM getting filled if you play to much with the iterative plots. Now that's fixed.
- Since the last exercise is based on the xdmf output, I reverted to fault xdmf, also because this does not brings much improvement.
- I've added one more step in the rupture speed analysis, in which we decrease initial stress to suppress supershear.

**Comment by Thomas-Ulrich** [comment]
Also there is this piece of code that compute sigma_d, sigma_n and R from the stress tensor.
Could be integrated to the notebook for commenting on the initial stress and supershear rupture.
```python
import sympy as sp
import numpy as np

# Define symbolic variables
a, sigma_max, dip, mu_s, mu_d = sp.symbols("a sigma_max dip mu_s mu_d")

# stress tensor
stress = sp.diag((1 + a) * sigma_max / 2, a * sigma_max, sigma_max)
# normal and along dip unit vectors
u_n = sp.Matrix([0, -sp.sin(dip), sp.cos(dip)])
udip = sp.Matrix([0, sp.cos(dip), sp.sin(dip)])

# tractions
traction = stress * u_n

sigma_n = traction.dot(u_n)
print("sigma_n:", sigma_n)
sigma_d = traction.dot(udip)
print("sigma_d:", sigma_d)

R = (sigma_d - mu_d * sigma_n) / ((mu_s - mu_d) * sigma_n)
print("relative prestress ration R:", R)

params = {dip: 60 * np.pi / 180, a: 0.3495, mu_s: 0.54, mu_d: 0.1}
print(
    f"relative prestress ration R evaluated with {params} (nucleation):", R.subs(params)
)

params = {dip: 60 * np.pi / 180, a: 0.3495, mu_s: 0.7, mu_d: 0.1}
print(
    f"relative prestress ration R evaluated with {params} (outside nucleation):",
    R.subs(params),
)

print("sigma_n:", sigma_n.subs(params))
print("sigma_d:", sigma_d.subs(params))
```

File: tpv13/parameters.par [hunk 2]
```diff
@@ -34,6 +36,7 @@ printtimeinterval_sec = 1.0                    ! Time interval at which output w
 OutputMask = 1 1 1 1 1 1 1 1 1 1 1             ! turn on and off fault outputs
 refinement_strategy = 2
 refinement = 1
+!vtkorder = 3
```
Comment (davschneller) [inline tpv13/parameters.par:39]: Commented out?

**Review by davschneller — APPROVED** [review]
LGTM from looking over the source.

(do we even need the `Dockerfile`, i.e. the Frontera container, at the moment? It's not built right now anyways)

File: tpv13/parameters.par (hunk 2 above)
Comment (Thomas-Ulrich) [reply tpv13/parameters.par:39]: yes, because I decided not to use, because this would require updating the paraview example below.

**Comment by Thomas-Ulrich** [comment]
> (do we even need the `Dockerfile`, i.e. the Frontera container, at the moment? It's not built right now anyways)

you mean, frontera does not need a specific file anymore and we can remove the file?

**Comment by davschneller** [comment]
> > (do we even need the `Dockerfile`, i.e. the Frontera container, at the moment? It's not built right now anyways)
> 
> you mean, frontera does not need a specific file anymore and we can remove the file?

It still does need a special one. However, our CI doesn't build it at the moment (that can be fixed quite easily though).

**Pull request merged**